crates/db_schema/src/source/community.rs

```diff
@@ -189,52 +188,53 @@ pub struct CommunityUpdateForm {
 #[cfg_attr(feature = "full", diesel(table_name = community_actions))]
 #[cfg_attr(feature = "full", diesel(primary_key(person_id, community_id)))]
 #[cfg_attr(feature = "full", diesel(check_for_backend(diesel::pg::Pg)))]
-pub struct CommunityModerator {
+#[cfg_attr(feature = "full", ts(export))]
+pub struct CommunityActions {
   pub community_id: CommunityId,
   pub person_id: PersonId,
-  #[cfg_attr(feature = "full", diesel(select_expression = community_actions::became_moderator.assume_not_null()))]
-  #[cfg_attr(feature = "full", diesel(select_expression_type = dsl::AssumeNotNull<community_actions::became_moderator>))]
-  pub published: DateTime<Utc>,
+  #[cfg_attr(feature = "full", ts(optional))]
+  /// When the community was followed.
+  pub followed: Option<DateTime<Utc>>,
```

Comment:
Can be `serde(ignore)` as follow state is already included below.

Reply:
I'd rather not have that one be a special case. Sometimes actions come with data, so that data gets its own column. Another exampled is `liked, like_score`. Clients might find the timestamp of the like useful, in addition to the like score. Same for followed.